test/README.md

```diff
@@ -20,41 +20,15 @@ and run the following tests in docker containers:
 ## End-to-end tests
 
 You can run e2e nightly tests locally by running
+
 ```
 make -j2 docker generator runner && ./build/generator -g 5 -d networks/nightly/ -p && ./run-multiple.sh networks/nightly/*-group*-*.toml
 ```
+
 from the root directory of the repository.
 
 Please refer to the [README.MD](e2e/README.md) in the `e2e` folder for more information.
 
-## `localnet` tests
```

Comment:
maybe instead of removing these, we can replace with commands to run it with the `e2e` runner using the `networks/simple.toml` which only starts 4 validator node